### PR TITLE
made changes to powershell commandlet

### PR DESCRIPTION
**Pull request by anwara000**
made changes to powershell commandlet

Original cmdlet was giving error

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/information-barriers-policies.md [hunk 1]
```diff
@@ -71,10 +71,11 @@ In addition to the [required licenses and permissions](information-barriers.md#r
 
 - Admin consent for information barriers in Microsoft Teams -  When your IB policies are in place, they can remove non-IB compliance users from Groups (i.e. Teams channels, which are based on groups). This configuration helps ensure your organization remains compliant with policies and regulations. Use the following procedure to enable information barrier policies to work as expected in Microsoft Teams.
 
-   1. Run the following PowerShell cmdlets:
+   1. Pre-requisite: Install Azure PowerShell from [here](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md:74]:
```suggestion
   1. Pre-requisite: [Install Azure PowerShell](https://docs.microsoft.com/powershell/azure/install-az-ps).
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md:74]:
```suggestion
   1. Pre-requisite: Install Azure PowerShell from [here](https://docs.microsoft.com/powershell/azure/install-az-ps).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@anwara-ms

File: microsoft-365/compliance/information-barriers-policies.md [hunk 2]
```diff
@@ -71,10 +71,11 @@ In addition to the [required licenses and permissions](information-barriers.md#r
...
+   2. Run the following PowerShell cmdlets:
 
       ```powershell
-      Connect-AzAccount 
+      Connect-AzAccount -Tenant "<yourtenantdomain.com>"  //e.g. Connect-AzAccount -Tenant "Contoso.onmicrosoft.com"
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md]:
```suggestion
      Connect-AzAccount -Tenant "<yourtenantdomain.com>", for example Connect-AzAccount -Tenant "Contoso.onmicrosoft.com"
```

**Comment by anwara000** [comment]
plz dont remove "//", as if you remove it, the line will become part of code that runs in powershell and will give error output.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by anwara000 — COMMENTED** [review]
all good from my end.

**Comment by anwara000** [comment]
I cant approve. Can someone approve and merge to production plz?

**Pull request merged**